src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const lines = sourceString.split(';');
```

Comment:
The variable name 'result' is too generic. Consider using a more descriptive name that reflects its purpose, such as 'styleObject' or 'cssProperties'.